bootstrap/test/clean.test.ts

```diff
@@ -0,0 +1,11 @@
+import {} from 'jest';
+import * as childProcess from 'child_process';
+import * as fs from 'fs';
+
+test('GET /test', async() => {
+    childProcess.spawnSync('./absolute', ['build_server'], { shell: true });
```

Comment:
that line doesn't work properly on Windows targets, so run ./absolute command as
childProcess.spawnSync(path.resolve('./absolute'), ['build_server'], { shell: true });